app/controllers/proposal/manage_collection_proposals_controller.rb

```diff
@@ -1,11 +1,12 @@
 module Proposal
+
   class ManageCollectionProposalsController < ManageMetadataController
     skip_before_action :provider_set?
 
     before_action :ensure_non_nasa_draft_permissions
 
     def show
-      authorize CollectionDraftProposal, policy_class: CollectionDraftProposalPolicy
+      authorize CollectionDraftProposal, policy_class: ManageCollectionProposalPolicy
```

Comment:
does :manage_collection_proposal or something like how subscriptions does this work here?

Reply:
good call, made the change